var/spack/repos/builtin/packages/doxygen/package.py

```diff
@@ -39,8 +39,12 @@ class Doxygen(CMakePackage):
 
     variant("mscgen", default=False, description="Build with support for code graphs from mscgen.")
 
+    tags = ["build-tools"]
```

Comment:
Nit: Is it a build or documentation tool?

I can understand why someone would want to flag the package as a build tool for query purposes so I'll approve as is.

Reply:
I noticed there are no `doc`-related tags.  You could start one if you want 😉 .

Fair point about how it fits in workflows.